source/content/guides/drupal-9-migration/05-drupal-9-to-pantheon.md

```diff
@@ -15,6 +15,24 @@ editpath: drupal-9-migration/05-drupal-9-to-pantheon.md
 
 In this doc, you'll migrate an existing Composer-managed Drupal 9 site from another platform to a new Drupal 9 site with Integrated Composer on Pantheon.
 
+## Unresolved Questions / TO-DOs
+
+- [How] Do we address nested docroot vs not-nested docroot?
```

Comment:
Composer-managed sites should almost always have a nested docroot. For now we could just make that a requirement. We should address the fact that the docroot name might be different on the source site. It's best to rename it to `web`, as Pantheon expects.

Reply:
Okay, note @EdwardAngert Greg thinks we should address the subject of whether the existing site had a nested docroot and what it was named.

@greg-1-anderson in my draft, I had settled on simply not mentioning it and referring to the old/existing site paths relative to the docroot